Fabric.Authorization.AccessControl/setup-dependencies.ps1

```diff
@@ -0,0 +1,40 @@
+#
+# setup-dependencies.ps1
+#
+
+$nl = [Environment]::NewLine
+$nodeVersion = "10.12.0"
+$angularVersion = "6.1.2"
+$typescriptVersion = "3.1.3"
+
+# remove node_modules
+Write-Output ""
+Write-Output "$nl----] Removing previous node modules [----$nl"
+Remove-Item .\\node_modules -Force -Recurse
+
+# uninstall angular/cli
+Write-Output "$nl----] Removing previous angular cli [----$nl"
+npm uninstall -g @angular/cli

```

Comment:
should we check for the existence of choco before we delete?  if for some reason choco is not installed and we remove things, how do we put them back?

Reply:
All set above.  I even install it for you too!